templates/juju-architecture.html

```diff
@@ -0,0 +1,268 @@
+{% extends 'how-juju-works.html' %}
+
+{% block title %}Juju architecture{% endblock %}
+{% block meta_keywords %}Software operators{% endblock %}
+{% block meta_copydoc %}https://docs.google.com/document/d/11t4Z5zMGhyMIUZZbp61iP_eybKFld0kidsviAJKrMd0/edit{% endblock meta_copydoc %}
+{% block page_class %}is-paper{% endblock %}
+
+{% set selected_tab='juju-architecture' %}
+
+{% block how_juju_works_content %}
+<section class="p-section">
+  <div class="row--50-50">
+    <div class="col">
+      <h1 class="p-heading--2">Juju high-level architecture</h1>
+    </div>
+    <div class="col">
+      <p class="p-heading--5">Juju provides a declarative, model-driven, and interactive way to install, provision, maintain, update, upgrade, and integrate applications on and across Kubernetes, Linux containers, virtual machines, and bare metal machines, on public or private cloud.</p>
+      <p>The following diagram provides an overview of the high level architecture of a Juju deployment and it describes the key components of the system. You can find a more detailed explanation of all the components you find in this page in the <a href="/docs/juju/reference">reference section</a> of the Juju documentation.</p>
+    </div>
+  </div>
+  <div class="u-fixed-width" style="padding-top: 2rem; padding-bottom: 1rem">
```

Comment:
In here we have both top and bottom with different values. Both likely could be shallow section (1.5rem).